public/templates/loop.php

```diff
@@ -1,7 +1,7 @@
 <li class="isotope-item <?php echo esc_attr( $term_class ); ?> all">
-<?php if( has_post_thumbnail( $isotope_loop->ID ) ) : ?>
+<?php if( has_post_thumbnail( $this->isotope_loop->ID ) ) : ?>
```

Comment:
Can you empty all params for has_post_thumbnail function?
As it runs inside a loop it does not need any post ID.